user-console/src/main/resources/org/pentaho/mantle/public/themes/ruby/mantleRuby.css

```diff
@@ -1761,4 +1761,48 @@ hr {
   display: none;
 }
 
+.pentaho-tab-bar{
```

Comment:
Here and in other modified rules in this file, the `{` should have a space before.